tests/unit/clients/test_postgres_client.py

```diff
@@ -27,3 +36,13 @@ def test_parsing_postgres_url(self, url, username, password, hostname, port, dat
         assert client.password == password
         assert client.port == port
         assert client.database == database
+        assert client.url_query == query
+
+    @pytest.mark.parametrize(
+        "env_value, expected", (("test", {"applicationName": "test"}), ("", None),)
+    )
+    def test_add_application_name_from_env(self, mocker, env_value, expected):
+        mocked_os = mocker.patch("tentaclio.clients.postgres_client.os")
```

Comment:
Maybe a good use case for monkey patching?

https://docs.pytest.org/en/stable/monkeypatch.html